docs/network/net_binding_plugins/passt.md

```diff
@@ -123,16 +111,6 @@ The sidecar sources can be found
 The relevant sidecar image needs to be accessible by the cluster and
 specified in the Kubevirt CR when registering the network binding plugin.
 
-### Feature Gate
```

Comment:
This section needs to be retained for backward compatibility with KubeVirt versions prior to v1.5, which are still within our support window. Additionally, this change appears unrelated to the optimizations described in the commit message.

Reply:
you might want to see conflicts about https://github.com/kubevirt/user-guide/pull/926

imo keep the PR focusing only on the optimization req, nothing else